app/lib/project/MetapolatorProject.js

```diff
@@ -626,6 +627,10 @@ define([
         }
     };
 
+    _p.saveAs = function(blob, filename) {
+        saveAs(blob, filename);
```

Comment:
This is a no-go. As I understand it, `filesaver` is expected to declare `saveAs`. But that is entirely implicit. You should receive `saveAs` as argument of the module funnction. It's bad to declare a dependency and not to use it, anyways. So if you are not going to use a dependency directly, you should still give it a name in the module closure arguments.

Anyways, filesaver does this:
```js
define([], function() {
    return saveAs;
});
```

so you can do this:

```
define(['filesaver'], function(saveAs){ 
/* use saveAs() here */
})

```